docs/content/Auth/Security-Context.md

```diff
@@ -103,10 +113,55 @@ SELECT
 LIMIT 10000
 ```
 
+## Using COMPILE_CONTEXT
+
+In the example below `user_id`, `company_id`, `sub` and `iat` will be injected
```

Comment:
@hassankhan Maybe we should say that it will accessible both in COMPILE_CONTEXT and SECURITY_CONTEXT and then use the callout to tell the difference?